test.js

```diff
@@ -1,9 +1,14 @@
 var tape = require('tape')
+var Nanobus = require('nanobus')
+require('jsdom-global')(null, { url: 'http://localhost/' })
+window.localStorage = window.localStorage || {} // See: yoshuawuyts/nanotiming#7
+window.requestAnimationFrame = window.requestAnimationFrame || function (cb) { process.nextTick(cb) }
```

Comment:
This looks a little tricky; perhaps using https://github.com/juliangruber/tape-run might make it easier by testing in an actual browser env?